.github/workflows/reusable-build-test-release.yml

```diff
@@ -763,14 +776,18 @@ jobs:
       addon-upload-path: ${{ steps.test-setup.outputs.addon-upload-path }}
       directory-path: ${{ steps.test-setup.outputs.directory-path }}
       s3-bucket: ${{ steps.test-setup.outputs.s3-bucket }}
+    env:
+      BUILD_NAME: ${{ needs.build.outputs.buildname }}
     steps:
       - uses: actions/checkout@v3
         with:
           submodules: recursive
+          token: ${{ secrets.GH_TOKEN_ADMIN }}
       - name: setup for test
         id: test-setup
         shell: bash
         run: |
+          sudo apt-get install -y crudini
```

Comment:
This was installed in the container image before?

Reply:
Yes - we need to use crudini so I am resolving this conv.